style.css

```diff
@@ -14,22 +14,18 @@ body {
     background-size: cover;
 }
 
-nav {
+header {
     box-sizing: border-box;
     margin: 0;
-    padding: 0.5rem 1rem;
+    padding: 0.5rem 1.5rem;
     display: flex;
     justify-content: space-between;
     position: absolute;
     top: 0;
     width: 100vw;
+    font-size: 0.8rem;
 }
 
-/* header {
-    margin-top: 3rem;
-    margin-bottom: 1rem;
-} */
-
 h1 {
     margin: 0;
     /* font-weight: 500; */
```

Comment:
[OPTIONAL] Kindly remove all unused code in order for you to have a cleaner codebase.

Reply:
This file is not valid. You are checking the main branch, not the development branch.